Examples/ArpSpoofing/main.cpp

```diff
@@ -14,59 +14,54 @@
 #include <Logger.h>
 #include <getopt.h>
 
-
-#define EXIT_WITH_ERROR(reason) do { \
-	printUsage(); \
-	std::cout << std::endl << "ERROR: " << reason << std::endl << std::endl; \
-	exit(1); \
-	} while(0)
-
-
-static struct option L3FwdOptions[] =
-{
-	{"interface",  required_argument, nullptr, 'i'},
-	{"victim", required_argument, nullptr, 'c'},
-	{"gateway", required_argument, nullptr, 'g'},
-	{"help", no_argument, nullptr, 'h'},
-	{"version", no_argument, nullptr, 'v'},
-	{nullptr, 0, nullptr, 0}
+#define EXIT_WITH_ERROR(reason)                                                                                        \
+	do                                                                                                                 \
+	{                                                                                                                  \
+		printUsage();                                                                                                  \
+		std::cout << std::endl << "ERROR: " << reason << std::endl << std::endl;                                       \
+		exit(1);                                                                                                       \
+	} while (0)
+
+static struct option L3FwdOptions[] = {
+	{ "interface", required_argument, nullptr, 'i' },
+    { "victim",    required_argument, nullptr, 'c' },
+	{ "gateway",   required_argument, nullptr, 'g' },
+    { "help",      no_argument,       nullptr, 'h' },
+	{ "version",   no_argument,       nullptr, 'v' },
+    { nullptr,     0,                 nullptr, 0   }
```

Comment:
@tigercosmos for some reason clang-format mixes tabs and spaces here, I have no idea why 🤷‍♂️ 
I tried different things but couldn't make it to work. Maybe you know?

It happens in other files also, like here: https://github.com/seladb/PcapPlusPlus/blob/6990c13f0b24192b1b9230cc23227406a08a3927/Examples/IcmpFileTransfer/Common.cpp#L21

But in others it doesn't happen: https://github.com/seladb/PcapPlusPlus/blob/6990c13f0b24192b1b9230cc23227406a08a3927/Examples/DNSResolver/main.cpp#L19

Reply:
Apparently in some of the files, changing the order of the arguments fixes the issue, this is very weird 😕

Reply:
@seladb  whenever you think clang-format is stupid, you can turn off the format for that part.

Reply:
It's concerning that we can't rely on `clang-format` and need to go over the code ourselves. This kind of beats the purpose of a code formatter 😕

Reply:
@seladb this kind of stuff rarely happens. We are formatting a bunch of code at once so there might be a few weird formatting. But overall, we will get the fruits after all formatting. :)